PC/HousingDerivative/PowerUnit.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HW_7.Accessories;
+
+namespace HW_7.PC.HousingDerivative
+{
+    public class PowerUnit : HousingElements
+    {
+        public string? Name { get; set; }
+        public string? Countru { get; set; }
+        public int Prise { get; set; }
+
+        public override void Get()
+        {
+            Console.WriteLine("Power unit:");
```

Comment:
Every each of them doing kind of the same console output - make a separate method for such a logic